exchange/utils.go

```diff
@@ -36,7 +36,8 @@ func cleanOpenRTBRequests(ctx context.Context, orig *openrtb.BidRequest, usersyn
 	consent := extractConsent(orig)
 	if gdpr == 1 {
 		for bidder, bidReq := range requestsByBidder {
-			if ok, err := gDPR.PersonalInfoAllowed(ctx, bidder, consent); !ok && err == nil {
+			coreBidder := resolveBidder(bidder.String(), aliases)
```

Comment:
Similar to this comment: https://github.com/prebid/prebid-server/pull/826#issuecomment-471576897

You may also want to put `// Fixes #` comments over fixes where the code could go "either way." Like... if it's not immediately obvious which code is correct or why.

They're not as useful as  `// Prevents #` comments... but it are occasionally nice to have.  It's basically a signal that "hey... if you're refactoring this code, look at this issue and make sure you don't re-introduce it."